bin/kafka-console-consumer.sh

```diff
@@ -18,4 +18,17 @@ if [ "x$KAFKA_HEAP_OPTS" = "x" ]; then
     export KAFKA_HEAP_OPTS="-Xmx512M"
 fi
 
-exec $(dirname $0)/kafka-run-class.sh kafka.tools.ConsoleConsumer $@
+base_dir=$(dirname $0)/..
+
+if [ -z "$SCALA_VERSION" ]; then
+        SCALA_VERSION=2.10.6
+fi
+
+if [ -z "$SCALA_BINARY_VERSION" ]; then
+        SCALA_BINARY_VERSION=2.10
+fi
+
+JARPATH="$base_dir/core/build/dependant-libs-${SCALA_VERSION}/*.jar \
```

Comment:
Instead of configuring the JARPATH, could we not set something like:

PROJECT_NAME=core

And then use that to build the JARPATH in `kafka-run-class`?